main.go

```diff
@@ -99,10 +101,49 @@ func run(p params) error {
 		return errors.Wrap(err, "could not find failed tests")
 	}
 
-	err = j.createIssuesOrComments(failedTests)
+	issues, err := j.createIssuesOrComments(failedTests)
 	if err != nil {
 		return errors.Wrap(err, "could not create issues or comments")
 	}
+	return j.createHtml(issues)
+}
+
+//go:embed htmlOutput.html.tpl
+var htmlOutputTemplate string
+
+func (j junit2jira) createHtml(issues []*jira.Issue) error {
+	if j.htmlOutput == "" || len(issues) == 0 {
+		return nil
+	}
+	out := os.Stdout
+	if j.htmlOutput != "-" {
+		file, err := os.Create(j.htmlOutput)
+		if err != nil {
+			return fmt.Errorf("could not create file %s: %w", j.htmlOutput, err)
+		}
+		out = file
+		defer file.Close()
+	}
+	return j.renderHtml(issues, out)
+}
+
+type htmlData struct {
+	Issues  []*jira.Issue
+	JiraUrl string
+}
+
+func (j junit2jira) renderHtml(issues []*jira.Issue, out io.Writer) error {
+	t, err := template.New(j.htmlOutput).Parse(htmlOutputTemplate)
+	if err != nil {
+		return fmt.Errorf("could parse template: %w", err)
+	}
+	err = t.Execute(out, htmlData{
+		Issues:  issues,
+		JiraUrl: j.jiraUrl,
+	})
+	if err != nil {
+		return fmt.Errorf("could not render template %s: %w", j.htmlOutput, err)
```

Comment:
Will including `j.htmlOutput` in the error message be useful at all? AFAICT this is the name of the output filename, so might be somewhat confusing...